Automattic/jetpack
# VideoPress: Enhance behavior when deleting multiple videos

Fixes #27773

## Proposed changes:

* Adds a buffer of removed videos in order to refresh the page only after all delete actions have finished, avoiding multiple reloads with wrong upload count

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the VideoPress dashboard and upload multiple videos
* Delete multiple uploaded videos in quick succession
* Check that the page only reloads after deleting all videos, with no intermediate reloads of wrong upload count

Before | After
-|-
<video src="https://user-images.githubusercontent.com/8486249/205943092-d00cbbda-b87c-4376-b44b-89d801f63e51.mp4"> | <video src="https://user-images.githubusercontent.com/8486249/211898559-68c021a4-3d40-40ec-954f-9e8fad8723ed.mp4">
